lighthouse-core/audits/works-offline.js

```diff
@@ -41,14 +41,16 @@ class WorksOffline extends Audit {
    */
   static audit(artifacts) {
     let debugString;
+    const passed = artifacts.Offline === 200;
     if (!URL.equalWithExcludedFragments(artifacts.URL.initialUrl, artifacts.URL.finalUrl)) {
-      debugString = 'WARNING: You may be failing this check because your test URL ' +
-          `(${artifacts.URL.initialUrl}) was redirected to "${artifacts.URL.finalUrl}". ` +
-          'Try testing the second URL directly.';
+      const explanation = passed ? 'Your test URL' :
```

Comment:
had that originally actually but it makes some sense that we flag that we don't know for sure if the original URL they wanted can be loaded offline, but the message is still a little off, I'll adjust it